content/docs/tasks/traffic-management/egress.md

```diff
@@ -155,23 +162,30 @@ to set a timeout rule on calls to the httpbin.org service.
 ## Calling external services directly
 
 If you want to completely bypass Istio for a specific IP range,
-you can configure the source service's Envoy sidecar to prevent it from
+you can configure the Envoy sidecars to prevent them from
 [intercepting](/docs/concepts/traffic-management/request-routing/#communication-between-services)
-the external requests. This can be done using the `--includeIPRanges` option of
-[istioctl kube-inject](/docs/reference/commands/istioctl/#istioctl kube-inject)
-when starting the service.
+the external requests. This can be done by setting the `global.proxy.includeIPRanges` variable of
+[Helm](/docs/setup/kubernetes/helm-install/#customization-with-helm) and updating the `ConfigMap` _istio-sidecar-injector_ by `kubectl apply`. After _istio-sidecar-injector_ is updated, the value of `global.proxy.includeIPRanges` will affect all the future deployments of the application pods.
 
-The simplest way to use the `--includeIPRanges` option is to pass it the IP range(s)
+The simplest way to use the `global.proxy.includeIPRanges` variable is to pass it the IP range(s)
 used for internal cluster services, thereby excluding external IPs from being redirected
 to the sidecar proxy.
 The values used for internal IP range(s), however, depends on where your cluster is running.
-For example, with Minikube the range is 10.0.0.1&#47;24, so you would start the sleep service like this:
+For example, with Minikube the range is 10.0.0.1&#47;24, so you would update your `ConfigMap` _istio-sidecar-injector_ like this:
 
-```command
-$ kubectl apply -f <(istioctl kube-inject -f samples/sleep/sleep.yaml --includeIPRanges=10.0.0.1/24)
+```plain
+$ helm template install/kubernetes/helm/istio <the flags you used to install Istio> --set global.proxy.includeIPRanges="10.0.0.1/24" -x templates/sidecar-injector-configmap.yaml | kubectl apply -f -
```

Comment:
@vadimeisenbergibm One more comment, do we need to highlight `namespace` here? As the configMap is only visible to applications in a specified namespace.

Reply:
@gyliu513 I ask the user to use the same namespace as they used when they installed Istio. The users can use whichever namespace they want, I think.

Reply:
@vadimeisenbergibm Yes, so I think we need to highlight namespace here to tell customer to create the configMap to the namespace where the application want to be deployed.

Reply:
@gyliu513 I have added stressing that the same namespace must be used.

Reply:
how about when we install not using helm ?

Reply:
this is not using helm to install istio but use helm to generate a configMap.

Reply:
@zainul-ma 
> how about when we install not using helm ?

If you do not use helm to generate Istio installation files, you can edit your installation files.

Reply:
how to do that ? is there is a sample to do that ?

Reply:
@vadimeisenbergibm you mean installation files of istio 0.8 ?

Reply:
@zainul-ma yes, files like `istio.yaml`.

Reply:
sorry just make sure
is that file is 
```
istio-0.8.0/install/kubernetes/istio-demo.yaml
```
because i try istio-0.8.0 installation via that file like in website documentation

and if not using that , which file i should edit ?

i try follow installation doc in istio website
little bit different with istio 0.7.1

Reply:
the step is 
install helm  and make upgrade like 
https://istio.io/docs/setup/kubernetes/upgrading-istio/
so it's will be generated istio.yaml

@vadimeisenbergibm  so in istio.yaml which attribute i should edit ?

Reply:
@zainul-ma you should edit the following lines in _istio-demo.yaml_ 

```yaml
        [[ if (isset .ObjectMeta.Annotations "traffic.sidecar.istio.io/includeOutboundIPRanges") -]]
        - "[[ index .ObjectMeta.Annotations "traffic.sidecar.istio.io/includeOutboundIPRanges"  ]]"
        [[ else -]]
        - "*"
```

You should put a comma-separated list of CIDR blocks instead of `*`, e.g.:

```yaml
        [[ if (isset .ObjectMeta.Annotations "traffic.sidecar.istio.io/includeOutboundIPRanges") -]]
        - "[[ index .ObjectMeta.Annotations "traffic.sidecar.istio.io/includeOutboundIPRanges"  ]]"
        [[ else -]]
        - "172.30.0.0/16,172.20.0.0/16,10.10.10.0/24"
        [[ end -]]
        - "-x"
```

Reply:
thank you @vadimeisenbergibm  it's work now for me

Reply:
would this help with pod IPs?

Reply:
@vadimeisenbergibm Hi, Is this the service IP or the Pod IP. And can we just do a kubectl apply after the modification?

Reply:
@Parvathy-Geetha it is service ip, yes, you can apply it after your modification.

Reply:
@vadimeisenbergibm  The istio-sidecar-injector configmap is not reading ip ranges. If I set a CIDR range its ignoring, where as setting the specific ip address of the service works.
Working:
```
\"traffic.sidecar.istio.io/includeOutboundIPRanges\"  ]]\"\n    [[ else -]]\n
    \   - \"10.39.250.81\"\n    [[ end -]]\n  
```
Not Working:

```
\"traffic.sidecar.istio.io/includeOutboundIPRanges\"  ]]\"\n    [[ else -]]\n
    \   - \"10.39.240.0/20\"\n  
```